src/sidebar/test/permissions-test.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+var Permissions = require('../permissions');
+
+var userid = 'acct:flash@gord.on';
+
+describe('permissions', function () {
```

Comment:
Should this be `'src.sidebar.permissions'`? (As per new unwritten test naming convention)